nestkernel/event_delivery_manager.cpp

```diff
@@ -577,12 +596,26 @@ EventDeliveryManager::deliver_events( thread t )
 void
 EventDeliveryManager::gather_events( bool done )
 {
+  // IMPORTANT: Ensure that gather_events(..) is called from a single thread and
+  //            NOT from a parallel OpenMP region!!!
```

Comment:
I looked into this: Unfortunately, gather_events(..) is called from within a parallel region, although only by one thread, which is enforced by "omp single" and "omp master". However, omp_in_parallel() and also omp_get_num_threads() refer to the superordinate parallel region and don't perform as expected for this reason.
Anyhow, the problem that gather_events(..) should be called only by a single thread exists regardless of the StopWatch introduced here. In this way, the problem is independant from this specific pull request. But of course, it would be good to find a better solution than to just indicate this requirement not at all or just by a comment.